frontend/src/components/MapView/Layers/BoundaryDropdown.tsx

```diff
@@ -308,23 +302,36 @@ export function SimpleBoundaryDropdown({
     4: styles.menuItem3,
   };
 
-  // It's important for this to be another component, since the Select component
-  // acts on the `value` prop, which we need to hide from <Select /> since this isn't a menu item.
-  const out = (
+  const MenuItemRenderer = ({
+    index,
+    style,
+  }: {
+    index: number;
+    style: any;
+  }) => {
+    const area = flattenedAreaList[index];
+    return (
+      <MenuItem
+        classes={{ root: clsName[(area.level - rootLevel) as number] }}
+        key={area.adminCode}
+        value={area.adminCode}
+        style={style}
+      >
+        {area.label}
+      </MenuItem>
+    );
+  };
```

Comment:
Not sure we need to create sub-component for that. I think it could be easier to write it directly line 388 (when it is called)